charts/prometheus-pushgateway/templates/deployment.yaml

```diff
@@ -24,6 +24,7 @@ spec:
     spec:
       serviceAccountName: {{ template "prometheus-pushgateway.serviceAccountName" . }}
       containers:
+{{ toYaml .Values.containers | indent 8}}      
```

Comment:
imho should be name "extraContainers".
also schould have an example in values.yaml.

Reply:
No problem. I tried to be as close a possible from the naming used in the prometheus-operator. But I'll do as you said